mobile-app/cypress/e2e/functional/wallet/portfolio/convert/convert.spec.ts

```diff
@@ -40,10 +40,11 @@ context("Wallet - Convert DFI", () => {
       "aria-disabled"
     );
     cy.getByTestID("source_balance").contains(19.9);
-    cy.getByTestID("convert_source").contains("UTXO");
-    cy.getByTestID("convert_target").contains("Token");
+    cy.getByTestID("convert_token_button_FROM_display_symbol").contains("UTXO");
+    cy.getByTestID("convert_token_button_TO_display_symbol").contains("Token");
     cy.getByTestID("convert_input").type("1");
-    cy.getByTestID("convert_input_clear_button").click();
+    cy.getByTestID("convert_input").clear();
+    // cy.getByTestID("convert_input_clear_button").click();
```

Comment:
what is this comment for?

Reply:
removed comment was for the older ui design check not needed now